packages/@spectrum-icons/illustrations/src/File.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Illustration, IllustrationPropsWithoutChildren } from "@react-spectrum/icon";
+
+export default function File(props: IllustrationPropsWithoutChildren) {
+  return (
+    <Illustration {...props}>
+      <svg>
+        <path fill="var(--spectrum-global-color-static-white)" d="M6 31.503a2.503 2.503 0 01-2.5-2.5v-26A2.503 2.503 0 016 .502h15.38a1.988 1.988 0 011.411.582l5.12 5.103a2.015 2.015 0 01.589 1.416v21.399a2.502 2.502 0 01-2.5 2.5z" />
```

Comment:
For fill color, maybe we want a css variable that updates with the theme

Reply:
probably `--spectrum-global-color-gray-50` and `--spectrum-global-color-gray-500` (not static)

Reply:
ahh of course, thanks!